units/crons.go

```diff
@@ -21,8 +25,30 @@ func StartCrons(ctx context.Context, remote, local amboy.Queue) error {
 		"opts":    opts,
 	})
 
-	// amboy.IntervalQueueOperation(ctx, remote, time.Minute, time.Now(), opts, amboy.GroupQueueOperation())
-	// amboy.IntervalQueueOperation(ctx, local, time.Minute, time.Now(), opts, amboy.GroupQueueOperation())
+	amboy.IntervalQueueOperation(ctx, remote, time.Minute, time.Now(), opts,
+		amboy.GroupQueueOperationFactory(PopulateCleanupOldLogDataJobs()))
+
+	amboy.IntervalQueueOperation(ctx, local, time.Minute, time.Now(), opts,
+		amboy.GroupQueueOperationFactory(PopulateCleanupOldLogDataJobs()))
 
 	return nil
 }
+
+// Queue Population Tasks
+
+func PopulateCleanupOldLogDataJobs() amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+
+		db := db.GetDatabase()
+		tests, err := logkeeper.GetOldTests(db, time.Now())
```

Comment:
time now? or threshold time?

Reply:
Not the threshold time. But John is right this parameter isn't actually necessary anymore.